Add SDK 15.5

jaimergp opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Closes #2

jaimergp committed:
Add SDK 15.5

jaimergp committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.11.12.35.56

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19292640816](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19292640816). Examine the logs at this URL for more detail.</sub>

h-vetinari commented:
While we're at it, we may want to add 26 as a deployment target too. I can add the corresponding SDK to ci-setup. That would be [26.1](https://github.com/alexey-lysiuk/macos-sdk/tree/main/MacOSX26.1.sdk) as of right now; since this is the bleeding edge, I'm also fine to wait for a year, when macOS 27 comes out, and the SDK should be at `.5` minor version again.

This illustrates an overall weakness of our current setup though. If someone sets
```yaml
c_stdlib_version:  # [osx]
  - 14.4           # [osx]
```
that should in principle be legal, but we have neither the osx-sysroot nor the SDK for such a minor version.

jaimergp commented:
We have been using SDKs packaged at https://github.com/alexey-lysiuk/macos-sdk/releases, where I can't find any of those versions you mention :/

Edit: found some at https://github.com/joseluisq/macosx-sdks/releases

h-vetinari commented:
> We have been using SDKs packaged at https://github.com/alexey-lysiuk/macos-sdk/releases, where I can't find any of those versions you mention :/

The osx-sysroot is in principle independent of the SDK. Also, since https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/404, we have the ability to use SDKs that are present in that repo but not tagged as a release.

jaimergp commented:
See https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/407 for more SDKs. Adding those here too in a bit.

jaimergp committed:
More versions

h-vetinari committed:
Merge remote-tracking branch 'upstream/main' into bump-15

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.30.19.55.26

Other tools:
- conda-build 25.11.1
- rattler-build 0.55.0
- rattler-build-conda-compat 1.4.10

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```, ```recipe/meta.yaml```) and found it was in an excellent condition.
